### ISSUE TITLE
Remote .m3u file - LiquidSoap assumes location of audio files

### ISSUE THREAD

**Issue by mikemackay** [savonet/liquidsoap#1362]
Hi,

I have a service running on a remotely hosted site (domain.com), with LiquidSoap running on my local network. LS pulls the .m3u file from domain.com, however, LS assumes any items inside the .m3u file are hosted remotely - in my case, they are hosted locally (SAN).

Example .m3u file:
```
#EXTM3U
annotate:liq_cue_in="0.000",liq_cross_duration="0.420",duration="249.820",liq_amplify="-11.500dB":/Music/Album/19 - The Cranberries - Zombie.mp3
```

Example .liq file:
```
#!/usr/local/bin/liquidsoap

security = single("/home/username/fallback.mp3")

myplaylist = playlist(mode="randomize", "https://domain.com/playlist.m3u")
myplaylist = fallback(track_sensitive=true, [request.queue(id="override"), myplaylist])
myplaylist = crossfade(duration=2.0, fade_out=2.0, fade_in=0.01, default=(fun(a,b)->add(normalize=false,([b, a]))), conservative=true, myplaylist)
radio = fallback(track_sensitive= false, [myplaylist, security])

# Stream it out
output.icecast(%mp3,
  host = "domain.com", port = 8000,
  password = "********", mount = "stream",
  radio, fallible=true
)
```

Would it be possible to tell LS that my audio files are local to the machine, and not on domain.com? There was support for file:/// and file:/ added to 1.4.3 but this does not appear to solve the issue.

Thanks :)

**Comment by toots** [comment]
Should be fixed now, thanks for reporting! We should support at least the following combination:

`http://host/playlist.m3u`:
```
#EXTM3U
annotate:liq_cue_in="0.000",liq_cross_duration="0.420",duration="249.820",liq_amplify="-11.500dB":/path/to/localfile
```

`http://host/playlist.m3u`:
```
#EXTM3U
annotate:liq_cue_in="0.000",liq_cross_duration="0.420",duration="249.820",liq_amplify="-11.500dB":http://otherhost/path
```

`http://host/playlist.m3u`:
```
#EXTM3U
annotate:liq_cue_in="0.000",liq_cross_duration="0.420",duration="249.820",liq_amplify="-11.500dB":/relative/uri
```

And more.

Thanks for reporting!

**Comment by toots** [comment]
I forgot to add: the fix will go out with the `1.4.4` release, which we're hoping to get our by the end of the year.

**Comment by toots** [comment]
Sorry, the changes cause a pretty bad regression, I've had to revert them. I'll revisit this for liquidsoap 2.0
